pkgs/development/compilers/gcc/builder.sh

```diff
@@ -241,7 +241,7 @@ postInstall() {
     # More dependencies with the previous gcc or some libs (gccbug stores the build command line)
     rm -rf $out/bin/gccbug
 
-    if type "patchelf"; then
+    if [ -n "${dontPatchELF}" ] && type "patchelf"; then
```

Comment:
this should be `-z` i think?

Reply:
Good point... that probably means that this isn't being set properly on the static builds! I'll look into it.

Reply:
I changed this locally and am re-running the build.

Reply:
Yeah, so adding `assert staticCompiler;` to this expression fails, I guess that isn't acutally set in `static.nix`. I could add an override for GCC, but there's the added complication of it being used in the stdenv, and being wrapped by `wrapCC`. @matthewbauer do you have any advice on how I could cause this variable to be set when building `pkgsStatic.gcc`?

Reply:
yeah i think that will work

Reply:
static.nix should only apply to the things in pkgsStatic, so I think it will actually be okay here